### DISCUSSION TITLE
How to root and open up DWA0120?

### DISCUSSION THREAD

**Discussion by Echogolf123** [hack-technicolor/hack-technicolor#188] [Q&A]
I have a DWA0120 provided through UK ISP SSE and they appear to have kindly shut off the WiFi remotely before I switched my ISP. I'd like to open it back up and use it for whatever I can. I've seen [this](https://gist.github.com/chriscpritchard/db98167c0a1372ef16e131bfe9b76956) post and looked [here](https://hack-technicolor.readthedocs.io/en/stable/Repository/#what-to-do-if-your-firmware-is-not-listed-here) but can't see it listed. I'm happy to share _firmware dump and other stuff like firmware download history and firmware decryption keys_ but I will need instructions, preferably in English or carefully worded so that Google can translate. I believe I need a [_variant of the current #D strategy_](https://www.ilpuntotecnico.com/forum/index.php/topic,77981.msg246548.html#msg246548)?

What would you like me to do?
Thank you.

Product Vendor
Technicolor
Product Name
DWA0120
Software Version
18.3
Firmware Version
18.3.0278-2741042-20190614155643
Firmware OID
5d03a79b4f5d980688be59da
Bootloader Version
18.37.1138-0000000-20180919064610-fe2746878916b5342ae471d5c774a02a337aaac9
Hardware Version
VBNT-2

**Comment by Echogolf123** [comment]
With #D clash escaping method [here](https://www.ilpuntotecnico.com/forum/index.php/topic,77981.msg246548.html#msg246548) I was successful with engineer login however the `set uci.button.button....` command failed with invalid command.

**Reply by LuKePicci** [reply]
Yup, DWA0120 is rootable. I need some help from you in getting a few things grom that device in order to list it.

Let me convert this discussion into a proper Model Support Request issue and contact me on Telegram, I have to explain you how to get root with new clash escaping commands.

**Reply by LuKePicci** [reply]
Let's talk in #189
